clients/common/norm.cpp

```diff
@@ -226,13 +224,13 @@ double norm_check_general<rocblas_half>(char norm_type,
     // use triangle inequality ||a+b|| <= ||a|| + ||b|| to calculate upper limit for Frobenius norm
     // of strided batched matrix
 
-    std::unique_ptr<float[]> hCPU_float(new float[N * lda + (batch_count - 1) * stride_a]());
-    std::unique_ptr<float[]> hGPU_float(new float[N * lda + (batch_count - 1) * stride_a]());
-    for(int i_batch = 0; i_batch < batch_count; i_batch++)
```

Comment:
`rocblas_int` should be used instead of `int` in case `rocblas_int` is larger than `int`.